### ISSUE TITLE
Add try/except block to _get_city() method in HelpRequest model

### ISSUE THREAD

**Issue by melizeche** [melizeche/ayudapy#196]
**Quick fix:**

1. Move `city=''` before the `try` block
2. We need to catch when Line 105 fails(or the other lines), in the except block just `pass` or maybe logging
https://github.com/melizeche/ayudapy/blob/73b610fc0f40d7bc2c0bd6c9ccf8300c936fe29c/core/models.py#L105-L113

**Not so quick fix:** maybe another ticket for the future
Create an async job/queue that calls @cabupy reverse location api to update later(it's a really bad idea to make a network request in the `save() `method)
Error:
```
geopy.exc:GeocoderUnavailable

/core.views:request_form

Apr 13, '20 9:12 pm

Time

ayudapy(Python Application)

Instance

1

Count
3 similar errors to this in the last 24 hours
Next error
Error message

geopy.exc:GeocoderUnavailable: Service not available
Stack trace
Traceback (most recent call last):
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/newrelic/api/wsgi_application.py", line 666, in _nr_wsgi_application_wrapper_
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/newrelic/api/wsgi_application.py", line 194, in __init__
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/newrelic/api/wsgi_application.py", line 556, in _nr_wsgi_application_wrapper_
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/wsgi.py", line 141, in __call__
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/newrelic/hooks/framework_django.py", line 419, in _nr_wrapper_BaseHandler_get_response_
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 75, in get_response
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/utils/deprecation.py", line 94, in __call__
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/utils/deprecation.py", line 94, in __call__
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/utils/deprecation.py", line 94, in __call__
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/utils/deprecation.py", line 94, in __call__
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/utils/deprecation.py", line 94, in __call__
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/utils/deprecation.py", line 94, in __call__
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/utils/deprecation.py", line 94, in __call__
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/newrelic/hooks/framework_django.py", line 540, in wrapper
File "./core/views.py", line 43, in request_form
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/django/forms/models.py", line 458, in save
File "./core/models.py", line 118, in save
File "./core/models.py", line 105, in _get_city
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/geopy/geocoders/osm.py", line 468, in reverse
File "/home/marce/ayudapy/env/lib/python3.6/site-packages/geopy/geocoders/base.py", line 380, in _call_geocoder
```

**Comment by melizeche** [comment]
just realized that **we are** calling a network request in the `save()` method :facepalm: 
probably the best idea is to implement the reverse location service in our servers

**Comment by melizeche** [comment]
Fixed in 0e19505d741a6b505e0731851bcc72f5a820a029
